test/filename_matching.py

```diff
@@ -9,7 +9,7 @@
 def check_src_files_have_test():
     missing_test_files = []
 
-    excluded = ['mitmproxy/contrib/', 'mitmproxy/io/proto/',
+    excluded = ['mitmproxy/contrib/', 'mitmproxy/io/proto/', 'mitmproxy/proxy2/layers/http',
```

Comment:
👀

Reply:
Yes, the structure of the test files doesn't match the structure of the code. Coverage is pretty good, maybe we can eventually fix this.